### ISSUE TITLE
Skin changer not working?

### ISSUE THREAD

**Issue by NanoHunter** [Jire/Abendigo#19]
First of all, it is a really nice hack you created. I haven't tested out all of the plugins yet, but those I did worked extremely well and I just wanted to thank you for your amazing work.
The only exception being the skin changer, which is by no means necessary, but quite nice to have.
I have the plugin uncommented in the Abendigo.kt file and I hit the 'F1' Button multiple times when hack and game were running, but without success. Did I miss something? Or is it simply not working atm?
I left everything in the skin changer plugin on default. 
Some other random questions: Would adding the line Weapons.knife(x) work? Do you have any plans on implementing a sticker feature?

Thanks in advance and keep up the great work ^^

**Comment by Jire** [comment]
Did you build your project after you uncommented the line? You can do this by running _build.bat_ :smile:

**Comment by NanoHunter** [comment]
Yes, I did. Also tried two times with redowloaded versions, without succes. One I didn't edit, just uncommented, then build it, wasn't working either.

**Comment by davewolax** [comment]
I also noticed problem with skin changer. sometimes if I buy the weapon too late, the skin won't apply. It also doesn't apply to XM1014 and AWP. probably skin is not defined in the code for those.

**Comment by Jire** [comment]
@davewolax https://github.com/Jire/Abendigo/issues/29

**Comment by NanoHunter** [comment]
Plugin works now, but also noticed the error with primary / secondary apply
